blurb/blurb.py

```diff
@@ -701,7 +701,7 @@ def run(s):
     return process.stdout.decode('ascii')
 
 
-readme_re = re.compile(r"This is Python version [23]\.\d").match
+readme_re = re.compile(r"This is \w+ version \d\.\d+").match
```

Comment:
I think this change is good. I'm not sure we need to restrict the version to one digit in the major version, but we can always change that later if it becomes an actual problem.

Reply:
```suggestion
readme_re = re.compile(r"This is \w+ version \d+\.\d+").match
```

If we don't want to restrict the major version to one digit